### ISSUE TITLE
Store previous mappings

### ISSUE THREAD

**Issue by alex-r-bigelow** [Kitware/candela#227]
Per user studies, projects should store old mappings for previously-closed visualizations

For example:
1. The user chooses a bar chart and establishes the matchings
2. The user switches to a scatterplot (invalidating the matchings) and establishes new matchings
3. The user switches back to the bar chart... the previous matchings are lost

Alternative: if we support multiple visualizations, we may want to take a different approach

**Comment by ronichoudhury** [comment]
The per user storage of matchings could be done based on the names or types of columns (advanced feature) so that when they load a different dataset of the same shape they get the same mappings by default.

**Comment by ronichoudhury** [comment]
We could also provide a "clear mappings" button so the user has the option of reverting to the fundamental case when the system "remembers" old mappings.

**Comment by jeffbaumes** [comment]
@ronichoudhury I don't think we should remember mappings based on names of columns. It will sometimes do the wrong thing and it's better to do nothing than sometimes do the wrong thing. The clear mappings button is probably good to have in its own right, independent of how we store prior mappings.

**Comment by ronichoudhury** [comment]
@alex-r-bigelow independently convinced me of this earlier. These "remembered" mappings should simply be per-project, and based only on the mapping the user has instantiated.

**Comment by alex-r-bigelow** [comment]
I just talked with Brad, and he had a pretty cool idea in this vein... if we could store "bookmarks" as part of a project, we wouldn't (shouldn't?) have to automatically infer the users' stopping points. This would also allow multiple charts of the same type to be saved instead of just keeping the last one that the user had.
